packages/theme/src/cli/services/init.ts

```diff
@@ -1,5 +1,9 @@
 import {renderTasks} from '@shopify/cli-kit/node/ui'
-import {downloadGitRepository} from '@shopify/cli-kit/node/git'
+import {downloadGitRepository, getLatestTag} from '@shopify/cli-kit/node/git'
+import {inTemporaryDirectory} from '@shopify/cli-kit/node/fs'
+
+export const SKELETON_THEME_URL = 'https://github.com/Shopify/skeleton-theme.git'
```

Comment:
Is this a placeholder / is this the right repo? I might be missing some context about a release strategy here